PacketDotNet/IgmpV3MembershipQueryPacket.cs

```diff
@@ -67,7 +67,14 @@ public short Checksum
                                                                Header.Bytes);
 
         /// <summary>Gets or sets the IGMPv3 membership query max response time, in tenths of seconds.</summary>
-        public byte MaxResponseTime => CodeOrFloatingPointValue(MaxResponseCode);
+        public ushort MaxResponseTime
+        {
+            get => CodeOrFloatingPointValue(MaxResponseCode);

```

Comment:
Something I missed the previous one, can we rename this to GetCodeOrFloatingPointValue.
Also change to expression body style (similar to the get here). Also for the property below.

Reply:
Done!
And added missing return for floatValue < 128 in ConvertFloatingPointToCode().

Reply:
Can you also rename CodeOrFloatingPointValue (see above comment)?

Reply:
Done! 
And, also, corrected the return type on one of those methods.